### PR TITLE
Enable default argument splitting for room_join().

### LINKED ISSUE #1115: Broken argument parsing for channel names with whitespaces.
### I am...

* [x] Reporting a bug
* [ ] Suggesting a new feature
* [ ] Requesting help with running my bot
* [ ] Requesting help writing plugins
* [ ] Here about something else

### I am running...

* Errbot version: 5.1.2 (latest pip package)
* OS version: Windows 10
* Python version: 3.5
* Using a virtual environment: no

### Issue description

I think I've noticed some issues with the ErrBot command handing: Some channel names can contain whitespaces.  The command /room occupants "channel name" works and correctly calls query_room() with 'channel name', but /room join "channel name" calls query_room() with '"channel' - the string encapsulation didn't work.

### Steps to reproduce

1. Create a channel and include whitespace characters in its name.
2. Try out the above commands.

### PR DESCRIPTION
**Pull request by derAnfaenger**
Fixes #1115.

I hope you agree with the tradeoff here - one might not be able to enter a password with spaces directly anymore, but it's still possible with quotation marks.

### CODE REVIEW

File: errbot/core_plugins/chatRoom.py [hunk 1]
```diff
@@ -176,6 +175,7 @@ def room_occupants(self, message, args):
         Examples (IRC):
         !room occupants #example-room #another-example-room
         """
+        print("args: ", args)
```
Comment (zoni) [inline errbot/core_plugins/chatRoom.py]: Think you accidentally included this in your commit here. :smiley_cat:

Comment (derAnfaenger) [reply errbot/core_plugins/chatRoom.py]: :/ sorry, my bad. Will fix right away.

Comment (zoni) [reply errbot/core_plugins/chatRoom.py]: No worries, been there, done that. :smile:

**Comment by derAnfaenger** [comment]
@zoni changes made.

**Comment by zoni** [comment]
Thanks!

**Pull request merged**